process/smartContract/hooks/blockChainHook.go

```diff
@@ -771,6 +773,22 @@ func (bh *BlockChainHookImpl) EpochConfirmed(epoch uint32, _ uint64) {
 	}
 }
 
+// ExecuteSmartContractCallOnOtherVM on another VM
+func (bh *BlockChainHookImpl) ExecuteSmartContractCallOnOtherVM(input *vmcommon.ContractCallInput) (*vmcommon.VMOutput, error) {
+	if bh.vmContainer == nil {
```

Comment:
this should not be here. Make a new vmContainer on the constructor of blockchainHookImpl - and the next lines will return error, as they will not find the VM By sc address.

Reply:
done